src/bundle/Controller/User/UserResetPasswordController.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUiBundle\Controller\User;
+
+use eZ\Publish\Core\Base\Exceptions\NotFoundException;
+use EzSystems\EzPlatformAdminUi\Form\Factory\FormFactory;
+use EzSystems\EzPlatformAdminUi\Notification\NotificationHandlerInterface;
+use EzSystems\EzPlatformAdminUiBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use eZ\Publish\API\Repository\UserService;
+use eZ\Publish\API\Repository\PermissionResolver;
+use Symfony\Component\OptionsResolver\Exception\InvalidOptionsException;
+
+class UserResetPasswordController extends Controller
```

Comment:
I will put it into the discussion - I'm against creating so many controllers. If both "forgot password" and "reset password" are parts of a common feature I'd put them into one controller. It's very confusing later on when you need to change stuff in so many places. Number of controllers in AdminUI is getting out of hand already.

Let's hear from @alongosz, @Nattfarinn and @ViniTou though.